app/views/components/docs/autocomplete.yml

```diff
@@ -12,190 +14,176 @@ examples:
       name: autocomplete
       label:
         text: Select your country
-      options:
-        -
+      select:
+        options:
           -
-        -
-          - France
-          - fr
-        -
-          - Germany
-          - de
-        -
-          - United Kingdom
-          - uk
+            -
+          -
+            - France
+            - fr
+          -
+            - Germany
+            - de
+          -
+            - United Kingdom
+            - uk
 
   with_selected_value:
     data:
       id: autocomplete-selected
       name: autocomplete-selected
       label:
         text: Select your country
-      options:
-        -
-          - France
-          - fr
-        -
-          - Germany
-          - de
-        -
-          - United Kingdom
-          - uk
-      selected_options:
-          - de
+        bold: true
+      hint: Only a few countries are available
+      select:
+        options:
+          -
+            - France
+            - fr
+          -
+            - Germany
+            - de
+          -
+            - United Kingdom
+            - uk
+        selected: de
 
   with_error:
     data:
       name: autocomplete-with-error
       label:
         text: Autocomplete with error
-      options:
-        -
-          - France
-          - fr
-        -
-          - Germany
-          - de
-        -
-          - United Kingdom
-          - uk
+      select:
+        options:
+          -
+            - France
+            - fr
+          -
+            - Germany
+            - de
+          -
+            - United Kingdom
+            - uk
       error_items:
         - text: There is a problem with this input
 
-  multiselect:
+  select_multiple:
     data:
       id: autocomplete-multiselect
       name: autocomplete-multiselect
       label:
         text: Select your country
-      multiple: true
-      options:
-        -
-          - France
-          - fr
-        -
-          - Germany
-          - de
-        -
-          - United Kingdom
-          - uk
+      select:
+        multiple: true
+        options:
+          -
+            - France
+            - fr
+          -
+            - Germany
+            - de
+          -
+            - United Kingdom
+            - uk
+        selected:
+            - fr
+            - de
 
-  multiselect_with_selected_values:
+  autocomplete_narrow_width:
     data:
-      id: autocomplete-multiselect-selected
-      name: autocomplete-multiselect-selected
+      id: autocomplete-narrow
+      name: autocomplete-narrow
+      type: without-narrowing-results
```

Comment:
I don't think we're using this type anymore. We probably want to set this to
```
data_attributes: {
  "autocomplete-without-narrowing-results": true
},
```
especially since we don't have an example of how an enhanced select element works with this feature.